readme/Releases.md

```diff
@@ -86,4 +86,29 @@ The script will:
 During the merge it is imperative that the instructions of the script are followed stringently to not confuse our versioning scheme.
 
 
-
+## Release checklist
+
+To summarize all of the above information, the following checklist should be
+followed when publishing a release:
+
+* [ ] Update `CHANGELOG.md` with the latest changes.
+* [ ] After the `CHANGELOG.md` is fully updated, summarize the changes in
```

Comment:
Do the changes to `debian/changelog` have to be staged/commited?

Reply:
Yes. Actually, the changes to `debian/changelog` might have to be amended into the release commit because of the way the PPA picks up new releases. Not sure if there's a simpler way to include this. Will update the checklist to reflect this fact.